ui/tests/e2e/models.spec.js

```diff
@@ -0,0 +1,62 @@
+import { test, expect } from '@playwright/test';
+import path from 'path';
+
+const MODEL_URL = 'git://github.com/aws-controllers-k8s/apigatewayv2-controller/main/helm';
+const MODEL_NAME = 'test-model';
+const MODEL_DISPLAY_NAME = 'Test Model';
+const TAR_FILE_PATH = path.resolve('tests/e2e/assets/test.tar');
+const APP_URL = 'http://localhost:9081/';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(APP_URL);
+  await page.getByTestId('settings-button').getByRole('button').click();
+  await page.getByTestId('settings-tab-registry').click();
+});
+
+test('Create a Model', async ({ page }) => {
+  await page.getByTestId('create-model-button').click();
+
+  await page.getByRole('textbox', { name: 'Model Name' }).fill(MODEL_NAME);
+  await page.getByRole('textbox', { name: 'Model Display Name' }).fill(MODEL_DISPLAY_NAME);
+
+  await page.getByRole('button', { name: 'Next' }).click();
+  await page.getByRole('button', { name: 'Next' }).click();
+  await page.getByRole('button', { name: 'Next' }).click();
```

Comment:
can we assert on some intermediate steps here to ensure UI is behaving as anticpated.